photon.py

```diff
@@ -2,14 +2,14 @@
 # -*- coding: utf-8 -*-
 
 # Let's import what we need
+import argparse
 import os
 import sys
-import time
 import random
-import warnings
-import argparse
```

Comment:
Why did you re-arrange imports? Any specific reason?
Because I kept them in order according to their length so it looks less cluttered.
I have followed the same schema wherever possible.

Reply:
I did them by standard/3rd party in alphabetical order to quickly search for specific modules. If you disagree and think it makes it worse I totally understand :) its just how I prefer imports